3rdParty/EndianPortable/include/EndianPortable.h

```diff
@@ -67,6 +67,10 @@
 #elif defined(WIN32) || defined(WINx64)
 #include <winsock2.h>
  #if(BYTE_ORDER == LITTLE_ENDIAN)
+ #if defined(PCAPPP_MINGW_ENV)
```

Comment:
IMO PCAPPP_MINGW_ENV is a bit ambigous.

```
#ifndef htonll
  #define htonll(x) ((1==htonl(1)) ? (x) : (((uint64_t)htonl((x) & 0xFFFFFFFFUL)) << 32) | htonl((uint32_t)((x) >> 32)))
#endif // htonll
#ifndef ntohll
  #define ntohll(x) ((1==ntohl(1)) ? (x) : (((uint64_t)ntohl((x) & 0xFFFFFFFFUL)) << 32) | ntohl((uint32_t)((x) >> 32)))
#endif // ntohll
```

And also this should be defined out of BYTE_ORDER == LITTLE_ENDIAN no ?

Reply:
Updated, thanks